pocket_coffea/utils/stat/combine.py

```diff
@@ -104,10 +108,10 @@ def __init__(
         for process in self.mc_processes.values():
             for year in process.years:
                 if process.is_signal:
-                    self.process_id[f"{process.name}_{year}"] = id_signal
+                    self.process_id[f"{process.name}{self.process_suffix}"] = id_signal
```

Comment:
Similarly here (and in all occurrences below): should the key be `f"{process.name}{self.process_suffix}"` or `f"{process.name}_{self.process_suffix}"`? Maybe I would prefer the second so the user doesn't need to know that it should append a `_` to the `process_suffix` string.